packages/hdwallet-native/src/binance.ts

```diff
@@ -77,20 +77,21 @@ export function MixinNativeBinanceWallet<TBase extends core.Constructor<NativeHD
         const { privateKey } = util.getKeyPair(this.#wallet, msg.addressNList, "binance");
 
         const client = new BncClient("https://dex.binance.org"); // broadcast not used but available
-        client.chainId = msg.chain_id;
-        client.setAccountNumber(Number(msg.account_number) || undefined);
         await client.chooseNetwork("mainnet");
-        await client.setPrivateKey(privateKey, Number.isInteger(Number(msg.account_number)));
+        await client.setPrivateKey(privateKey);
```

Comment:
Kevin made a PR specifically to add the account number to fix the issue of signatures being different. Can you explain how this fixes the current issue by reverting that change?

Reply:
this offloads the account lookup to the sdk, but perhaps is not needed. was simply trying to rule out issues debugging

Reply:
ill add this back